src/Context/products-context.js

```diff
@@ -0,0 +1,17 @@
+import data from "../data";
+import { createContext, useContext, useState } from "react";
+
+const ProductContext = createContext(null)
+
+const ProductProvider = ({children}) =>{
+    const [prodArr,updateProdArr] = useState(data);
```

Comment:
Again, just a convention the second element of the array that `useState` is always a setter function & we usually give it a name that starts with `set` with the appended state name. It's the common convention & would help anyone reading your code to understand it better.